src/main/java/duke/TaskListError.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+public class TaskListError extends IndexOutOfBoundsException {
+    
+    private static String errorType1 = "Accessing out of bounds index in TaskList " +
```

Comment:
Perhaps avoid naming variables like var1, var2 and so on? Try using a short form of the type of error such as 'accessError' or 'deleteError'?